controllers/v1/sprint.go

```diff
@@ -35,12 +35,15 @@ func (ctrl SprintController) Routes(r *gin.RouterGroup) {
 func (ctrl SprintController) List(c *gin.Context) {
 	userID, _ := c.Get("userID")
 	retroID := c.Param("retroID")
+	after, _ := c.GetQuery("after")
+	perPage, _ := c.GetQuery("count")
```

Comment:
Can we use `GetInt` here?

Reply:
The [docs](https://github.com/gin-gonic/gin/blob/master/context.go#L219) say, 
```GetInt returns the value associated with the key as an integer.```

The usage of `GetInt` is to extract an integer from a key/value pair for the context which has been set using `Set`.

We use `GetQuery` here to extract a query param. As per my understanding, this cannot be accomplished by `GetInt`.